src/main/java/hudson/plugins/git/extensions/impl/ChangelogToBranch.java

```diff
@@ -9,7 +9,7 @@
 
 /**
  * This extension activates the alternative changelog computation,
- * where the changelog is calculated against a specified branch.
+ * where the changelog is calculated against a specified branch or tag.
```

Comment:
Technically, now it can work with any branch, tag or other revision syntax which supports being prefixed by "^".